src/generic/data/apiHooks.test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
```

Comment:
nit: I'm not entirely sure, but should this file be `.ts` instead of `.tsx`?

Reply:
We need `.tsx` here because we are using jsx syntax to render the hook:

```
const wrapper = ({ children }) => (
  <QueryClientProvider client={queryClient}>
    {children}
  </QueryClientProvider>
);
```

Reply:
Ah yes, got it, thanks!